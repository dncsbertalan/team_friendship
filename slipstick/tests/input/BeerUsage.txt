load testmap.txt
move Student_1 Room_1
move Student_1 Room_2
pick_up_item Beer_1
pick_up_item Beer_2
use_item Beer_1
list -in Student_1
exit
