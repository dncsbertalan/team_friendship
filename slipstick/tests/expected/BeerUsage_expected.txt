Successfully loaded testmap.txt
Moving Student_1 from MainHall to Room_1
Moving Student_1 from Room_1 to Room_2
Student_1 picked up Beer_1
Student_1 picked up Beer_2
Student_1 used Beer_1
Student_1 dropped Beer_2
Student_1's inventory: (empty)
